kong/clustering/tls.lua

```diff
@@ -226,7 +228,7 @@ function tls.validate_client_cert(kong_config, cp_cert, dp_cert_pem)
     return nil, err
   end
 
-  return true
+  return cert, nil
```

Comment:
nit: second `nil` return value is not necessary

```suggestion
  return cert
```

Reply:
I think this is pretty good because it keeps the number of returned values consistent. 😂